CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing to the project
+
+## PR process
+* When you open a PR, add a few reviewers. If the PR solves a GitHub issue, make sure
+  that the contributor who opened the issue is one of the reviewers.
+* If you do not see any progress on the PR over a couple of days, feel free to put a
+  comment tagging one or two users asking them to take the time to review your change.
+* The reviewer will perform the code review. If they are satisfied with the change, and
+  they feel like the change does not require a second pair of eyes, they will merge the
+  PR.
+* The PR maybe large enough that it may be worth getting a second opinion. In that case,
```

Comment:
"The PR maybe large enough that..." => "If the PR is large enough it may be..." or "The PR may be large enough that..."

Reply:
I've reworded the sentence to note that size is not the only factor to decide having 2 reviewers